### PR TITLE
CBL-4215: Fix broken Kotlin tests

### PR DESCRIPTION
**Pull request by bmeike**

### CODE REVIEW

File: common/main/java/com/couchbase/lite/Collection.java [hunk 1]
```diff
@@ -136,16 +136,19 @@ static Collection getDefaultCollection(@NonNull Database db) throws CouchbaseLit
...
     /**
-     * Get scope
+     * Return the collection name
      */
     @NonNull
-    public Scope getScope() { return new Scope(c4Collection.getScope(), db); }
+    public String getName() { return c4Collection.getName(); }
+
+    @NonNull
+    public String getFullName() { return c4Collection.getScope() + "." + c4Collection.getName(); }
```
Comment (pasin) [inline common/main/java/com/couchbase/lite/Collection.java:151]: Can it be non-public?

**Review by pasin — CHANGES_REQUESTED** [review]

File: common/main/java/com/couchbase/lite/Collection.java (hunk 1 above)
Comment (bmeike) [reply common/main/java/com/couchbase/lite/Collection.java:151]: Yes... but why?  You are about to request that it be public... That seems just so mean to our clients....

Comment (pasin) [reply common/main/java/com/couchbase/lite/Collection.java:151]: I knew I want to have it public for all but not for one. Until then, I suggested that it is non-public.

**Review by pasin — APPROVED** [review]

File: common/main/java/com/couchbase/lite/Collection.java (hunk 1 above)
Comment (bmeike) [reply common/main/java/com/couchbase/lite/Collection.java]: Done.  You are just a meany!!

Comment (pasin) [reply common/main/java/com/couchbase/lite/Collection.java]: Sorry for that. I need to keep the rule that we all jump at the same time if possible.

**Pull request merged**